extensions/DynamicPageList/DPLMain.php

```diff
@@ -2675,7 +2675,15 @@ public static function dynamicPageList( $input, $params, $parser, &$bReset, $cal
             //PAGE LINK
             $sTitleText = $title->getText();
             if ($bShowNamespace) $sTitleText = $title->getPrefixedText();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)

[project-id]: wikia:app
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)

[project-id]: wikia:app